libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -962,7 +962,9 @@ AP_InertialSensor::init(uint16_t loop_rate)
         if (!notch.params.enabled() && !fft_enabled) {
             continue;
         }
-        notch.calculated_notch_freq_hz[0] = notch.params.center_freq_hz();
```

Comment:
for 4.5.x I want to remove all these lines and leave the value as zero, meaning unitialised.
The calculated_notch_freq_hz array is an array of observed frequencies from ESCs. We have not observed this frequency in the init, so we should not be filling in a false value
we should handle the zero in the filter